content/roadmap/en.md

```diff
@@ -1,3 +1,115 @@
 # Roadmap
 
-TBD
+## Release coming soon 
+
+In ordered to improve the convenience of Zwin, we are planning to release the following features soon.
```

Comment:
Should add a nuance that we recognize the importance of these features, like "following critical features"

Reply:
same in Japanese